src/main/java/org/opentripplanner/routing/impl/OrcaFareServiceImpl.java

```diff
@@ -185,6 +218,18 @@ private static RideType classify(Route routeData, String tripId) {
             } else if (tripId.contains("west")) {
                 rideType = RideType.KITSAP_TRANSIT_FAST_FERRY_WESTBOUND;
             }
+        } else if (rideType == RideType.SOUND_TRANSIT &&
```

Comment:
Create unit tests to cover 1-Line, S Line and N Line (should these two have a hyphen like 1-Line?) cases.

Reply:
For some reason there is no hyphen in them in the GTFS... Perhaps we should just strip special characters like that in case the GTFS changes.